easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'mpi4py'
+version = '1.3.1'
+
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
```

Comment:
nope, `$MPICC` is used below, usempi means defining `$CC=$MPICC`, which is not needed here